scripts/bump.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+echo "Bumping... "
+message='🚀🔖 release v%s'
+
+if [[ -z "$1" ]] ; then
+      npm version --message "$message" patch
+      exit 0
```

Comment:
check indentation

Reply:
also, this could probably be refactored to have only a single call to `npm version` by just settings `$1` to `patch` here.